sqlmesh/core/snapshot/evaluator.py

```diff
@@ -1825,3 +1826,11 @@ def _check_destructive_schema_change(
         raise SQLMeshError(
             f"{warning_msg} To allow this, change the model's `on_destructive_change` setting to `warn` or `allow` or include it in the plan's `--allow-destructive-model` option."
         )
+
+
+def _check_table_db_is_physical_schema(table_name: str, physical_schema: str) -> None:
```

Comment:
`is_physical_schema` or `is_in_physical_schema`?

Reply:
I think `is_physical_schema` is correct right? We are checking if the physical schema matches the database name. One thing that might cause confusion is that we are mixing the terms `schema` and `db` to mean the same thing (because SQLGlot refers to it as `db` while SQLMesh refers to it here as `schema`).

Reply:
Actually yeah, you're right. This throws me off all the time, some things are `catalog.db.table`, some are `catalog.schema.table` and elsewhere might be closer to `db.schema.table`